docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -7,74 +7,221 @@
+some limitations.
+
+Changed entities will only be identified if their time field 
+has also been updated and falls within the range of the action to check for 
+changes. This has been designed in principal for, and is suited to the use case 
+where new data is given a timestamp for the time of ingest. 
+
+If the indices that fall within the scope of the source index pattern are 
+removed, for example when deleting historical time-based indices, then the 
+composite aggregation performed in consecutive checkpoint processing will search 
+over different source data, and entities that only existed in the deleted index 
+will not be removed from the {dataframe} destination index.
+
+Depending on your use case, you may wish to recreate the {dataframe-transform} 
+entirely after deletions. Alternatively, if your use case is tolerant to 
+historical archiving, you may wish to include a max ingest timestamp in your 
+aggregation. This will allow you to exclude results that have not been recently 
+updated when viewing the {dataframe} destination index.
+
 
 [float]
-[[df-aggresponse-limitations]]
-=== Aggregation responses may be incompatible with destination index mappings
+[[df-deletion-limitations]]
+=== Deleting a {dataframe-transform} does not delete the {dataframe} destination 
+index or {kib} index pattern
+
+When deleting a {dataframe-transform} using `DELETE _data_frame/transforms/index` 
+neither the {dataframe} destination index nor the {kib} index pattern, should 
+one have been created, are deleted. These objects must be deleted separately.
+
+[float]
+[[df-aggregation-page-limitations]]
+=== Handling dynamic adjustment of aggregation page size
+
+During the development of {dataframe-transforms}, control was favoured over 
+performance. In the design considerations, it is preferred for the 
+{dataframe} to take longer to complete quietly in the background rather than to 
+finish quickly and take precedence in resource consumption.
+
+Composite aggregations are well suited for high cardinality data enabling 
+pagination through results. If a parent 
+{ref}/circuit-breaker.html[circuit breaker] memory exception occurs when 
+performing the composite aggregated search then we try again reducing the number 
+of buckets requested. This circuit breaker is calculated based upon all activity 
+within the cluster, not just activity from {dataframe-transforms}, so it 
+therefore may only be a temporary resource availability issue.
+
+For a batch {dataframe}, the number of buckets requested is only ever adjusted 
+downwards. The lowering of value may result in a longer duration for the 
+transform checkpoint to complete. For {cdataframes}, the number of 
+buckets requested is reset back to its default at the start of every checkpoint 
+and it is possible for circuit breaker exceptions to occur repeatedly in the 
+{es} logs. 
+
+The initial value for `composite.size` is 500 (which is configurable using 
```

Comment:
sounds confusing to me, can we describe what it does with less technical detail?

data frame transform retrieves data in batches which means it calculates several buckets at once. Per default this is 500 buckets per search/index operation. The default can be changed using `max_page_search_size` and the minimum value is 10. ...

I think we can explain everything without mentioning composite aggs. I do not want to hide it, I just think it's not necessary to reference it.